code/modules/clothing/accessories/accessory.dm

```diff
@@ -1059,3 +1052,58 @@
 	desc = "An expensive watch with a black band wrapped around your wrist. Snazzy."
 	icon_state = "wristwatch_black"
 	item_state = "wristwatch_black"
+
+/*Waistcoat*/
```

Comment:
Well fuck, this kinda ruins/conflicts with my Helltaker waistcoat. While them being an accesory makes sense so you can wear them below a suit jacket and I'm all for it, I'm gonna have to refactor my own as it's more of a vest (with pockets) than an undersuit waistcoat then